pkg/supportbundle/supportbundle.go

```diff
@@ -312,7 +312,12 @@ func CreateSupportBundleAnalysis(appID string, archivePath string, bundle *types
 	if err != nil {
 		logger.Errorf("Failed to check if cluster is kurl: %v", err)
 	}
-	analyzer.Spec.Analyzers = append(analyzer.Spec.Analyzers, getDefaultAnalyzers(isKurl)...)
+
+	defaultAnalyzers, err := getDefaultAnalyzers(isKurl, foundApp.IsAirgap)
+	if err != nil {
```

Comment:
should you wrap and log the error here? it looks like everything else in this function does